infra/partner-mainnet/scripts/example.txt

```diff
@@ -0,0 +1,4 @@
+multichain-account-sk-mainnet-0=foo
```

Comment:
also will this have any danger of including \n or space? in the secret?

Reply:
I accounted for that in the script itself, so as long as they follow this format (<secret-name>=<sec-val> ) it will work.

Reply:
ah nice!